sys-apps/tuned/tuned-2.13.0.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit python-single-r1 systemd xdg-utils
+
+DESCRIPTION="Daemon for monitoring and adaptive tuning of system devices"
+HOMEPAGE="https://fedorahosted.org/tuned/"
+SRC_URI="https://github.com/redhat-performance/tuned/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
```

Comment:
keyword also for `~x86`

Reply:
I've copied tuned-2.10.0 which already dropped it. ced5658b20ceb2b385067f4dc3c5546eb50929c3 states:
> 2) drop x86 keyword due to dev-python/python-linux-procfs